### PR TITLE
Added PR labeler

### PR DESCRIPTION
**Pull request by NandVinchhi**
This is a PR labeler implemented using Github Actions.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+documentation:
+- "**/*.md"
+
+continuous_integration:
```
Comment (logankilpatrick) [inline .github/labeler.yml]: not that this string "continuous_integration" corresponds to the labels for pull request found here: https://github.com/waralex/Dashboards.jl/labels

You need to make sure you use labels that actually exist or open an issue on the repo and request that the maintainer adds the PR labels you want so the labeler can work as expected.

Comment (NandVinchhi) [reply .github/labeler.yml]: Sorry, I misunderstood the process. I will update soon.

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+documentation:
+- "./*.md"
+- "**/*.md"
+
+enhancement:
+- "src/*.jl"
+- "test/*"
+- "test/**/*"
+- "./*"
```
Comment (logankilpatrick) [inline .github/labeler.yml:16]: I would remove this

**Comment by logankilpatrick** [comment]
You should open an issue and request a "testing" label. The test file path should not fall under the enhancement label.

**Comment by NandVinchhi** [comment]
ok

**Pull request merged**